compiler/tests/fail/warning/load_constant_warning.jazz

```diff
@@ -0,0 +1,10 @@
+export fn foo(reg u32 x) -> reg u32 {
+
+  while (x < 10) {
+    x = x + 1;
+  }
+
+  x = x;
+  return x;
+
+}
```

Comment:
I would move the file to a new subdir `risc-v`, because it is only on this arch that there is a warning.